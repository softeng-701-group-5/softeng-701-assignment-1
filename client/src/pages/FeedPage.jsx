import React from 'react';
import {
  AppBar,
  Typography,
  Toolbar,
  Container,
  Paper,
  makeStyles,
} from '@material-ui/core';

const useStyles = makeStyles(theme => ({
  root: {
    flexGrow: 1,
  },
  menuButton: {
    marginRight: theme.spacing(2),
  },
  title: {
    flexGrow: 1,
  },
}));

export const FeedPage = () => {
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <AppBar position="static">
        <Toolbar>
          <Typography className={classes.title} variant="h6">
            Feedr
          </Typography>
        </Toolbar>
      </AppBar>

      <Container>
        <Paper>
          <p>welcome to the feed page</p>
        </Paper>
      </Container>
    </div>
  );
};
