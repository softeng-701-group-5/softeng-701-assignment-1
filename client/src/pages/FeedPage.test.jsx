import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import { mount } from 'enzyme';
import { FeedPage } from './FeedPage';

describe('<FeedPage />', () => {
  let subject;

  beforeEach(() => {
    subject = mount(
      <Router>
        <FeedPage />
      </Router>
    );
  });

  afterEach(() => {
    subject.unmount();
  });

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <Router>
        <FeedPage />
      </Router>,
      div
    );
  });
});
