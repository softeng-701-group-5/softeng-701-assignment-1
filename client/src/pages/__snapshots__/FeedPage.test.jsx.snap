// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FeedPage /> matches snapshot 1`] = `
<div
  className="makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiGrid-root makeStyles-appBar-3 MuiGrid-align-items-xs-center MuiGrid-justify-xs-space-between"
    >
      <div
        className="makeStyles-appBarContents-4"
      >
        <h6
          className="MuiTypography-root makeStyles-title-2 MuiTypography-h6"
        >
          Feedr
        </h6>
        <div
          className="makeStyles-search-184"
        >
          <div
            className="makeStyles-searchIcon-185"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
              />
            </svg>
          </div>
          <div
            className="MuiInputBase-root makeStyles-inputRoot-186 MuiInputBase-fullWidth"
            onClick={[Function]}
          >
            <input
              aria-label="search"
              className="MuiInputBase-input makeStyles-inputInput-187 MuiInputBase-inputTypeSearch"
              id="search"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Search Content"
              type="search"
            />
          </div>
        </div>
      </div>
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiIconButton-root makeStyles-logoutButton-9 MuiIconButton-colorInherit"
        href="/"
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M10.09 15.59L11.5 17l5-5-5-5-1.41 1.41L12.67 11H3v2h9.67l-2.58 2.59zM19 3H5c-1.11 0-2 .9-2 2v4h2V5h14v14H5v-4H3v4c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
            />
          </svg>
        </span>
      </a>
    </div>
  </header>
  <div
    className="MuiContainer-root makeStyles-container-5 MuiContainer-maxWidthLg"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-justify-xs-center"
    >
      <div
        className="MuiCircularProgress-root makeStyles-loader-10 MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
        role="progressbar"
        style={
          Object {
            "height": 40,
            "width": 40,
          }
        }
      >
        <svg
          className="MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
            cx={44}
            cy={44}
            fill="none"
            r={20.2}
            strokeWidth={3.6}
            style={Object {}}
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
